src/ontodia/widgets/classTree/leaf.tsx

```diff
@@ -84,6 +87,14 @@ export class Leaf extends React.Component<LeafProps, State> {
                             </span>
                         ) : null}
                     </a>
+                    {creatableClasses.get(node.model.id) ? (
+                        <div className={`${LEAF_CLASS}__create ontodia-btn-group ontodia-btn-group-xs`}>
+                            <button className='ontodia-btn ontodia-btn-default'
+                                onClick={this.create}>
+                                create
```

Comment:
What do you think about using an icon instead of the text?

Reply:
I tried different icons and wasn't able to find anything decent-looking. Maybe you can suggest something?

Reply:
How about [this](https://fontawesome.com/icons/plus?style=solid) or [this](https://fontawesome.com/icons/plus-square?style=solid) icons?